pkg/scheduler/core/equivalence_cache_test.go

```diff
@@ -316,28 +466,30 @@ func TestPredicateWithECache(t *testing.T) {
 	for _, test := range tests {
 		ecache := NewEquivalenceCache()
 		// set cached item to equivalence cache
-		ecache.UpdateCachedPredicateItem(
+		ecache.mu.Lock()
```

Comment:
It certainly does not hurt to add the locks, but do we actually need them in these unit-tests?

Reply:
I'm more comfortable leaving it as-is, and as you said, it doesn't hurt. (=